packages/react-chat/src/components/SystemResponse/SystemMessage.tsx

```diff
@@ -44,28 +44,31 @@ export interface SystemMessageProps {
   feedback?: FeedbackProps | undefined;
 }
 
-const SystemMessage: React.FC<SystemMessageProps> = ({ avatar, feedback, timestamp, message, withImage }) => {
+const SystemMessage: React.FC<SystemMessageProps> = ({ avatar, feedback, timestamp, message, withImage, children }) => {
   const containerRef = useRef<HTMLDivElement>(null);
   const controlsRef = useRef<HTMLSpanElement>(null);
 
-  if (message.type === MessageType.END) {
+  if (!children && message?.type === MessageType.END) {
```

Comment:
do not render `EndState` if `children` provided (likely would not be an `end` message anyway unless extremely customized